packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -142,6 +142,10 @@ function addEventListener(node, event, handler, capture) {
     },
   };
 }
+// mock return value of addEventListener with noop; used as default
```

Comment:
Ideally, I would stick to the same API as we us in the rest of the codebase. I'm not aware we use the `remove` structure anymore: https://github.com/mui-org/material-ui/blob/50c656b27817c14f5afee8ebef5345a504c7662b/packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js#L321

Reply:
I just refactored the existing pattern but I'll change it.